src/fs.rs

```diff
@@ -1,13 +1,42 @@
+extern crate rand;
```

Comment:
extern calls should be in `src/main.rs` (I think?)

Reply:
fixed